enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/files/AbstractRequireFiles.java

```diff
@@ -93,21 +93,36 @@ private void fail(List<File> failures) throws EnforcerRuleException {
 
         StringBuilder buf = new StringBuilder();
         if (message != null) {
-            buf.append(message + System.lineSeparator());
+            buf.append(message + '\n');
```

Comment:
I'm not sure how it will be worked on windows console .... 
Can anybody confirm ...
@michael-o @Bukama

Reply:
Agree, this isn't right

Reply:
![image](https://github.com/user-attachments/assets/3b179f0c-62db-4a2b-9bd3-17272579d9e9)


at least jshell handles them the same.

![image](https://github.com/user-attachments/assets/be1714da-a839-4c44-a65f-2b52802f6678)

so does cmd